testcases/network/rpc/rpc-tirpc/tests_pack/rpc_svc_1/rpc_svc_1.c

```diff
@@ -103,10 +103,10 @@ char *simplePing(union u_argument *inVar, SVCXPRT * transp)
 char *svc_getcaller_test(union u_argument *inVar, SVCXPRT * transp)
 {
 	//In this function we test svc_getcaller function basically (simple call)
-	struct sockaddr_in *sa = NULL;
+	struct sockaddr *sa = NULL;
```

Comment:
NOTE: IPv6 is not used at all, try to run `rpc_test.sh` with `-6` and it fails, therefore this does not make sense unless you add `rpc_test.sh` support.